1365.py

```diff
@@ -0,0 +1,16 @@
+class Solution:
+    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
+        sorted_nums = sorted(nums)
+        ans = []
+        
+        for num in nums:
+            count = 0
+            
+            for sorted_num in sorted_nums:
+                if num > sorted_num:
+                    count += 1
+            
+            ans.append(count)
+            count = 0
```

Comment:
Same issue as [this](https://github.com/woke7/leetcode-python/pull/21#discussion_r606711085)